src/NServiceBus.Core/Configure_Obsolete.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus
+{
+    using NServiceBus.Config;
+
+    public partial class Configure
+    {
+        [ObsoleteEx(Message = "Remove when IWantToRunWhenConfigurationIsComplete is obsoleted.", RemoveInVersion = "7.0")]
+        void RegisterWantToRunWhenConfigurationIsCompleteAsInstancePerCall()
```

Comment:
well you have obsoleted IWantToRunWhenConfigurationIsComplete  in 7. as per https://github.com/Fody/Obsolete#steptype it means "Treat as error" will be 6.

So once we go 6 no one can use IWantToRunWhenConfigurationIsComplete   in v6. 

so if no one can use IWantToRunWhenConfigurationIsComplete   then there is no point registering them or running them.